controllers/grid/users/reviewer/form/ReviewerForm.inc.php

```diff
@@ -241,12 +242,16 @@ function fetch($request) {
 			'reviewDueDate' => __('reviewer.submission.reviewDueDate'),
 			'submissionReviewUrl' => __('common.url'),
 			'reviewerUserName' => __('user.username'),
+			'contextName' => $context->getLocalizedName(),
```

Comment:
I need some guidance in how I should properly pass these variables along through the `MailTemplate` class.

Take `editorialContactSignature` as an example. In `initData()`, it is assigned to the `SubmissionMailTemplate` object `$template` using `assignParams()`, but the `$template` object is not passed along entirely to the form; only its body is passed as `personalMessage`.
https://github.com/pkp/pkp-lib/blob/cb6fa16ad663618fe2133dc6dec556bb0e8535fa/controllers/grid/users/reviewer/form/ReviewerForm.inc.php#L206-L211

Therefore, during `readInputData()` and `execute()`, there seems to be no way to retrieve `editorialContactSignature`.

Is the following appropriate?
- use `$this->setData()` to supply `$editorialContactSignature` to the form
- add `$editorialContactSignature` as a hidden input to the `reviewerFormFooter.tpl`
- retrieve it back in `readInputData()` to make it availabe to `execute()`